Task 1/snake-water-gun.py

```diff
@@ -0,0 +1,164 @@
+ # Exercise - 5
```

Comment:
You might want to change the folder structure.

I'd suggest you put it in `Task 3` folder, and mention the sample input/output (& maybe some screenshots if you wish 😄 )